sdk/@launchdarkly/launchdarkly_flutter_observability/example/lib/main.dart

```diff
@@ -27,7 +26,18 @@ void main() {
           // If using android studio the `additional run args` option can include the correct --dart-define.
           CredentialSource.fromEnvironment(),
           AutoEnvAttributes.enabled,
-          plugins: [ObservabilityPlugin()],
+          plugins: [
+            ObservabilityPlugin(
+              applicationName: 'test-application',
+              // This could be a semantic version of a git commit hash.
+              // This demonstrates how to use an environment variable to set the hash.
+              // flutter build --dart-define GIT_SHA=$(git rev-parse HEAD) --dart-define LAUNCHDARKLY_MOBILE_KEY=<my-mobile-key>
+              applicationVersion: const String.fromEnvironment(
```

Comment:
What isn't nice is if you forget this const, then it doesn't work and it doesn't error. Leading to a lot of confused lost time. https://github.com/flutter/flutter/issues/55870

Reply:
:sadpanda: